src/js/components/DataTable/Resizer.js

```diff
@@ -12,55 +12,68 @@ const ResizerBox = styled(Box)`
   cursor: col-resize;
 `;
 
-class Resizer extends Component {
-  state = {};
+const Resizer = ({ onResize, property, theme }) => {
+  const [state , setState] = useState({
+    start: undefined,
+    width: undefined,
+    event: undefined,
+  });
```

Comment:
Can we please separate those from one state to multiple different states?

Reply:
Fixed!